### ISSUE TITLE
Option to prevent user from adding points in edit modus.

### ISSUE THREAD

**Issue by Eschon** [Leaflet/Leaflet.draw#420]
As the title says I would like to be able to prevent the user from adding additional points to a polygon when editing.

Is there already a proper way to do this or can this be added as a feature?

**Comment by SirenHound** [comment]
I don't know what the proper way to do it would be, but a quick solution is to rewrite the _createMiddleMarker function with an empty one.

**Comment by Eschon** [comment]
Yeah that would work but I have two editable maps in my page and it should be possible for one and not possible for the other so I would need to use two Leaflet.draw files, which I don't like.

**Comment by ddproxy** [comment]
@Eschon You can clone Leaflet.draw and overload the _createMiddleMarker function in the cloned version.

I intend on pushing editing functions off to Leaflet.Editable by 0.4 - so we should touch base on 0.4 to see if it's possible there.

**Comment by Eschon** [comment]
@ddproxy 
Thanks for the tip, I'll probably have to go back to the project where I ran into this issue soon, so I'll try that then.
